aw-watcher-bash-preexec.sh

```diff
@@ -0,0 +1,35 @@
+export PATH=$PATH:~/.local/bin/
+
+CURRENT_SHELL=$(ps -p "$$" | tail -n 1 | awk '{ print $4 }')
```

Comment:
I wouldn't recommend using upper-case only variable names, as these could be confused with in-built ones. (`$CURRENT_SHELL` now gets displayed when I write `$` and tab through all the possibilities. Without knowing this code, I'd guess that it is in-built and 99% safe to use)

Reply:
Even if it was lower-case you would still find it when you do $+tab since the file is sourced, it does not care about lower/upper case. We should probably do something to avoid polluting variables to the shell later.

Still though you are correct, it is better to use lower case in these scenarios so I did that now.